src/numbuf/thirdparty/download_thirdparty.sh

```diff
@@ -8,7 +8,5 @@ set -e
 TP_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 
 if [ ! -d $TP_DIR/arrow ]; then
-  git clone https://github.com/pcmoritz/arrow.git "$TP_DIR/arrow"
+  git clone https://github.com/apache/arrow/ "$TP_DIR/arrow"
```

Comment:
nice to finally be able to do this :) of course we should add a specific commit.